crates/bevy_ui/src/node_bundles.rs

```diff
@@ -34,22 +32,18 @@ pub struct NodeBundle {
     pub border_color: BorderColor,
     /// Whether this node should block interaction with lower nodes
     pub focus_policy: FocusPolicy,
-    /// The transform of the node
-    ///
-    /// This field is automatically managed by the UI layout system.
-    /// To alter the position of the `NodeBundle`, use the properties of the [`Style`] component.
-    pub transform: Transform,
-    /// The global transform of the node
-    ///
-    /// This field is automatically managed by the UI layout system.
-    /// To alter the position of the `NodeBundle`, use the properties of the [`Style`] component.
-    pub global_transform: GlobalTransform,
     /// Describes the visibility properties of the node
     pub visibility: Visibility,
-    /// Algorithmically-computed indication of whether an entity is visible and should be extracted for rendering
-    pub computed_visibility: ComputedVisibility,
```

Comment:
I don't think `ComputedVisibility` and `Visibility` are separated in a bundle anywhere in Bevy's code, it is a bit surprising to do it here.

Reply:
Separating `Visibility` and `ComputedVisibility` is kind of the point of this change. Users don't need to set `ComputedVisibility` at bundle creation, so its named field is just confusing and gets in the way.

I don't know if this is an ideal approach really. There wouldn't be much sense in using this pattern in most of the rest of Bevy as nothing else has bundles as large as the UI does or uses as many internally managed components afaik.